lib/experimental/fonts/font-library/font-library.php

```diff
@@ -187,3 +187,60 @@ function wp_get_font_dir( $defaults = array() ) {
 		return apply_filters( 'font_dir', $defaults );
 	}
 }
+
+// @core-merge: Filters should go in `src/wp-includes/default-filters.php`,
+// functions in a general file for font library.
+if ( ! function_exists( '_wp_deleted_font_family' ) ) {
```

Comment:
I'm guessing this is what the GH action is complaining about:

```suggestion
if ( ! function_exists( '_wp_delete_font_family' ) ) {
```